tcpdump/build.gradle

```diff
@@ -13,8 +13,6 @@ android {
     defaultConfig {
         minSdkVersion 26
         targetSdkVersion 29
-        versionCode 1
-        versionName '1.0'
```

Comment:
Why would your remove version information?
Doesn't it help the overall build system?

Reply:
It's only needed for the actual application gradles. Library submodules don't need them.